build/build.go

```diff
@@ -56,6 +56,8 @@ type artifact struct {
 	Dest string `yaml:"dest"`
 }
 
+// should be in-sync with driver/manifest.buildManifest
```

Comment:
Is there a practical way for us to detect when they get out of sync? Even if we still have to fix regressions manually, it'd be nice if CI could fail to let us know that we need to.

Reply:
No, as long as both are unexported. But we can move the `buildManifest` to `manifest` package and this package will import it.

Reply:
I have a better idea - it can be in the `internal` package, so it won't become our public API.

Reply:
Done